clusterctl/CONTRIBUTING.md

```diff
@@ -1,3 +1,26 @@
 # Contributing Guidelines
 
-1. Follow the [Getting Started]((https://github.com/kubernetes-sigs/cluster-api/blob/master/cluster-api/clusterctl/README.md)) steps to create a cluster.
\ No newline at end of file
+1. Follow the [Getting Started]((https://github.com/kubernetes-sigs/cluster-api/blob/master/cluster-api/clusterctl/README.md)) steps to create a cluster.
+
+# Development
+
+Before submitting an PR you should run the unit and integration tests. Instructions for doing so are given in the [Testing](#Testing) section.
+
+## Testing
```

Comment:
are there any prereqs for running the tests?
might be useful to list them out.

i think some of our unit tests requires etcd to be running on the local machine.
for integration tests, do they need gcloud installed?